--- origsrc/wxruby-1.9.10/swig/classes/include/wxDisplay.h	2009-02-04 09:48:14.000000000 -0600
+++ src/wxruby-1.9.10/swig/classes/include/wxDisplay.h	2009-02-04 22:41:45.003750000 -0600
@@ -17,10 +17,12 @@ public:
     wxRect GetGeometry() const;
     wxString GetName() const;
     bool IsPrimary() const;
+#if wxUSE_DISPLAY
     wxArrayVideoModes GetModes(const wxVideoMode& mode = wxDefaultVideoMode) const;
     wxVideoMode GetCurrentMode() const;
     bool ChangeMode(const wxVideoMode& mode = wxDefaultVideoMode);
     void ResetMode();
+#endif
     virtual ~wxDisplay();
 };
 
