--- origsrc/wxruby-1.9.7/rake/rakecygwin.rb	2008-05-10 05:01:08.000000000 -0500
+++ src/wxruby-1.9.7/rake/rakecygwin.rb	2008-05-22 07:23:30.562500000 -0500
@@ -6,6 +6,8 @@
 #############################
 # platform-dependent settings
 
-use_wx_config
+require 'rake/rakelinux'
 
 $extra_ld_flags = '-shared'
+
+$extra_libs += File.join(Config::CONFIG['libdir'], Config::CONFIG['LIBRUBY'])
--- origsrc/wxruby-1.9.7/rake/rakeunixish.rb	2008-05-10 05:01:08.000000000 -0500
+++ src/wxruby-1.9.7/rake/rakeunixish.rb	2008-05-22 07:28:36.046875000 -0500
@@ -139,9 +139,7 @@
 # monolithic build (in which case, the link deps are the same). If not
 # monolithic, need to add wxmedia to the list of required libs
 if not $excluded_classes.include?('MediaCtrl') 
-  if wx_config('--linkdeps std') != wx_config('--linkdeps std,media')
     libs_str << ',media'
-  end
 end
 
 # Set the final list of libs to be used
--- origsrc/wxruby-1.9.7/swig/classes/include/wxStyledTextCtrl.h	2008-05-10 05:01:21.000000000 -0500
+++ src/wxruby-1.9.7/swig/classes/include/wxStyledTextCtrl.h	2008-05-22 07:07:08.687500000 -0500
@@ -1791,7 +1791,7 @@
 
 //----------------------------------------------------------------------
 
-class WXDLLIMPEXP_STC wxStyledTextCtrl : public wxControl {
+class wxStyledTextCtrl : public wxControl {
 public:
 
 
